x/stakeibc/client/cli/query_controller_balances_test.go

```diff
@@ -0,0 +1,164 @@
+package cli_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	clitestutil "github.com/cosmos/cosmos-sdk/testutil/cli"
+	"github.com/stretchr/testify/require"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/Stride-Labs/stride/testutil/network"
+	"github.com/Stride-Labs/stride/testutil/nullify"
+	"github.com/Stride-Labs/stride/x/stakeibc/client/cli"
+    "github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+// Prevent strconv unused error
+var _ = strconv.IntSize
```

Comment:
What does this line do? does the linter not pick up on the strconv references below?

Reply:
not sure, it was scaffolded by ignite

Reply:
I removed it and the proj builds OK